Signal/src/view controllers/SignalTabBarController.m

```diff
@@ -33,9 +39,9 @@ - (void)viewDidLoad {
 
 - (void)viewDidAppear:(BOOL)animated{
     [super viewDidAppear:animated];
-    
-    if (![TSAccountManager isRegistered]){
-        [self performSegueWithIdentifier:@"showSignupFlow" sender:self];
+   
+    if (![Environment.preferences getIsMigratingToVersion2Dot0] && ![TSAccountManager isRegistered] ){
```

Comment:
this gives us time to finishing the migration process before starting registration. in the event there is an error with the migration (e.g. the server is down or client's net is down) we should discuss error condition. right now no error handling